index.html

```diff
@@ -1052,6 +1097,160 @@ <h4 id="Abilities">Abilities</h4>
             Other abilities can be used regardless of weapon type (see Types of Attack Roll
             section).
          </p>
+         <h4 id="Legendary Abilities">Legendary Abilities</h4>
+         <p>
+           You can unlock a legendary ability at level 1, and an additional one every 5 levels.  
+           These give a significant ability or passive to your character.
+           Some can be taken multiple times.
+         </p>
+         <table class="center">
+            <tr>
+               <td><strong>Name</strong>
+               </td>
+               <td><strong>Desc</strong>
+               </td>
+               <td><strong>Requirements</strong>
+               </td>
+            </tr>
+            <tr>
+               <td>Legendary Endurance
+               </td>
+               <td>
+                  Increase your max HP by additional CON value. This additional HP is recalculated as normal if 
+                  your CON value changes.
+               </td>
+               <td>Can be taken up to 3 times. You may only take either Legendary Endurance or Legendary Power
+                  abilities a total of 3 times. Such as taking Legendary Endurance 3 times and not taking Legendary Power (or vice versa),
+                  or 2 Legendary Endurance and 1 Legendary Power (or vice versa).
+               </td>
+            </tr>
+            <tr>
+               <td>Legendary Power
+               </td>
+               <td>
+                  Light / Medium / Heavy values increase by 1 / 2 / 3 respectively.
+               </td>
+               <td>Can be taken up to 3 times. You may only take either Legendary Endurance or Legendary Power
+                  abilities a total of 3 times. Such as taking Legendary Endurance 3 times and not taking Legendary Power (or vice versa),
+                  or 2 Legendary Endurance and 1 Legendary Power (or vice versa).
+               </td>
+            </tr>
+            <tr>
+               <td>
+                  Legendary Empowered Resistance Drain
+               </td>
+               <td>
+                  <p>
+                  Upgrade to Empowered Resistance Drain. When you use Empowered Resistance Drain, the number
```

Comment:
This feels kinda bad - a high cost 1AP ability to drain some resistance points and extra damage.  Empowered resistance drain never felt great, but it didn't matter because it was something you always had.  This still doesn't feel great, plus comes with arguably the biggest opportunity costs of Power/Health increases.